### DISCUSSION TITLE
Ghidra logo in SVG format

### DISCUSSION THREAD

**Discussion by agatti** [NationalSecurityAgency/ghidra#3709] [Q&A] [answered]
Hi there,

  is there a version of the Ghidra logo in SVG format out there (under an appropriately permissive usage licence)?  I'm working on a Sleigh-oriented plugin for IntelliJ and for icons and the like I'd rather use scaled down SVG images than fixed-size bitmaps when possible.

Thanks!

**Comment by ryanmkurtz** [comment, marked as answer]
Will this work for you?

[GHIDRA_FAV150.zip](https://github.com/NationalSecurityAgency/ghidra/files/7667409/GHIDRA_FAV150.zip)

**Comment by agatti** [comment]
That's awesome, thanks!  Just to be sure, this is under the same usage licence as Ghidra itself (Apache 2.0), right?

**Reply by ryanmkurtz** [reply]
> That's awesome, thanks! Just to be sure, this is under the same usage licence as Ghidra itself (Apache 2.0), right?

Yep!

**Reply by agatti** [reply]
Excellent, thanks!
